src/main/kotlin/com/msg/gcms/domain/attendance/service/impl/ClubAttendanceStatusExcelServiceImpl.kt

```diff
@@ -0,0 +1,95 @@
+package com.msg.gcms.domain.attendance.service.impl
+
+import com.msg.gcms.domain.attendance.domain.entity.Schedule
+import com.msg.gcms.domain.attendance.repository.AttendanceRepository
+import com.msg.gcms.domain.attendance.repository.ScheduleRepository
+import com.msg.gcms.domain.attendance.service.ClubAttendanceStatusExcelService
+import com.msg.gcms.domain.club.domain.entity.Club
+import com.msg.gcms.global.annotation.ServiceWithReadOnlyTransaction
+import com.msg.gcms.global.util.ExcelUtil
+import org.apache.poi.ss.usermodel.*
+import org.apache.poi.xssf.usermodel.XSSFCellStyle
+import org.apache.poi.xssf.usermodel.XSSFSheet
+import org.apache.poi.xssf.usermodel.XSSFWorkbook
+import java.io.ByteArrayOutputStream
+import java.time.LocalDate
+
+@ServiceWithReadOnlyTransaction
+class ClubAttendanceStatusExcelServiceImpl(
+    private val scheduleRepository: ScheduleRepository,
+    private val attendanceRepository: AttendanceRepository
+
+
+): ClubAttendanceStatusExcelService {
+    override fun execute(currentDate: LocalDate): ByteArray {
+        val schedule = scheduleRepository.findAllByDate(currentDate)
+
+        val workBook = XSSFWorkbook()
+
+        val font = workBook.createFont()
+        font.fontName = "Arial"
+        font.fontHeightInPoints = 11
+
+        schedule.map {
```

Comment:
```suggestion
        schedule.forEach {
```
요거는 Element를 변환하는 map보다 forEach가 더 적절해보여요